### PR TITLE
block people from viewing and editing 1.0 apps

### PR DESCRIPTION
**Pull request by dannyroberts**
this is an easy-to-revert step that can validate upfront our assumption that this is ok

we can later gut everything underneath that is still around to support 1.0 apps

If you visit a 1.0 app, you'll see this page:

screen shot 2016-04-20 at 3 33 16 pm

### CODE REVIEW

**Comment by dannyroberts** [comment]
@amsagoff just checking you're onboard with this plan. This only disables 1.0 app building. Here were your thoughts on March 14:


Amelia Sagoff
ITS OFFICIAL
we can :skull: 1.x submissions

Danny Roberts
Oh dear lord the day has finally come!
:sun_small_cloud:
So I can just remove all 1.x related code with no concerns of backward compatibility?
both app manager and submission handling?
let me just do some last minute ass-covering
and then yes

Danny Roberts
:+1:

Amelia Sagoff
ok
i am 99% sure the answer is yes
if you can confirm that wvindia is the only project submitting 1.x over the last 3 months, then we can go ahead
and remove both the app manager and submission code
(kevin is working with interrupt on an admin report that will show version numbers, so dont feel like you need to pull it now unless you're super excited about this)

Danny Roberts
ok
well this is great news. Thanks Amelia!

Amelia Sagoff
np :excellent:

File: corehq/apps/app_manager/views/view_generic.py [hunk 1]
```diff
@@ -71,6 +71,12 @@ def view_generic(request, domain, app_id=None, module_id=None, form_id=None,
     except ModuleNotFoundException:
         return bail(request, domain, app_id)
 
+    if app and app.application_version == '1.0':
+        return render(request, 'app_manager/no_longer_supported.html', {
```
Comment (orangejenny) [inline corehq/apps/app_manager/views/view_generic.py]: Might be a good place for a soft_assert, so we get notified even if they don't contact support.

**Comment by czue** [comment]
:skull:

**Comment by amsagoff** [comment]
:skull: :grey_exclamation:

File: corehq/apps/app_manager/views/view_generic.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/apps/app_manager/views/view_generic.py]: Fair point! I'll add that

**Comment by dannyroberts** [comment]
✅ Thanks @orangejenny done

**Pull request merged**